plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/FlexDeployPreferencesPanel.java

```diff
@@ -164,9 +171,12 @@ private void updateControls() {
 
   private void applyPreferences(FlexDeployPreferences preferences) {
     useValuesButton.setSelection(preferences.getUseDeploymentPreferences());
-    gaeConfigFolderText.setText(preferences.getAppEngineDirectory());
```

Comment:
I have no idea how this happened, I must have deleted it accidentally. Thanks for catching it!